src/web/js/trove/chart-lib.js

```diff
@@ -1286,6 +1293,7 @@
 
     // ASSERT: if we're using custom images, *every* series will have idx 3 defined
     const hasImage = combined.every(p => get(p, 'ps').filter(p => p[3]).length > 0);
+    let dotChartP = false;
```

Comment:
curious about why you're not setting this directly, ie `const dotChartP = get(p, 'dot-chart')`

Reply:
`p` is not lexically available there. It is introduced by the function that's mapped across `combined`

Reply:
Rewrote and done